docs/server/security.md

```diff
@@ -134,11 +134,10 @@ The default JWT `ClaimMapper` needs a public key to perform validation of tokens
 You can use the default as an example to build your own for translating a caller's authorization information from other formats and conventions into Temporal `Claims`.
 
 ```go
+// authorization.NewDefaultJWTClaimMapper(provider TokenKeyProvider, cfg *config.Authorization, logger log.Logger)
```

Comment:
Why this commented out line?

Reply:
Actually, to make this block a valid Go code mostly.
I actually don't know why do we have snippets like this in our docs:
```
claimMapper := authorization.NewDefaultJWTClaimMapper(provider, cfg, logger)
```
What is `provider` here? What is the type of this specific `cfg`?
It would be much more useful to have a method signature instead.

Reply:
Never mind - I missed that this is just signature of the method for reference.

Reply:
I would add something like this to the line before the code snippet that shows the API call:

> To get an instance of the default ClaimMapper, call [`NewDefaultJWTClaimMapper`](https://github.com/temporalio/temporal/blob/master/common/authorization/default_jwt_claim_mapper.go) and provide it with an instance of a [`TokenKeyProvider`](https://github.com/temporalio/temporal/blob/master/common/authorization/token_key_provider.go#L34) (described below), a pointer to an [`Authorization`](https://github.com/temporalio/temporal/blob/master/common/config/config.go#L479) config, and a [logger](https://github.com/temporalio/temporal/blob/master/common/log/interface.go#L42).